wallet/src/de/schildbach/wallet/ui/dashpay/PlatformRepo.kt

```diff
@@ -475,13 +475,15 @@ class PlatformRepo private constructor(val walletApplication: WalletApplication)
 
     suspend fun getNotificationCount(date: Long): Int {
         var count = 0
-        // Developer Mode Feature
-        if (walletApplication.configuration.developerMode && shouldShowAlert()) {
```

Comment:
`shouldShowAlert()` is not called anymore in the changed code and the method seems to be not used, is that ok?

Reply:
Thanks for catching this.  `shouldShowAlert()` needs to be called because we allow the user to dismiss the alert to never be shown again.  By removing `shouldShowAlert()` the user would always see that alert.

Reply:
Actually, `shouldShowAlert()` looks at other things to determine if that the alert should be displayed on the screen.